backend/server/repositories/bucket_repository.ts

```diff
@@ -1,11 +1,11 @@
-const Bucket = require('../models/bucket');
-const Promise = require('bluebird');
-const logger = require('../logger');
-const PageUtils = require('../utils/page_utils');
+import logger from '../logger';
+import { BucketModel } from '../models/bucket';
+import * as Promise from 'bluebird';
```

Comment:
Like this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Or something else?